src/formatting.ts

```diff
@@ -218,24 +218,109 @@ function toggleList() {
 }
 
 function toggleListSingleLine(doc: TextDocument, line: number, wsEdit: WorkspaceEdit) {
+    const empty = "";
+    const dash = "- ";
+    const star = "* ";
+    const plus = "+ ";
+    const num = "1. ";
+    const numClosingParetheses = "1) ";
+    const simpleListStart = [dash, star, plus]
+    const defaultMarkerArray = [dash, star, plus, num, numClosingParetheses];
```

Comment:
I know, naming seems a bit casual in this module due to some historical problems.

For new code, please think of names carefully.

Reply:
Thanks, you are a good reviewer. Could you please give me more hint about those variable name?